test/judges/test-github-events.rb

```diff
@@ -55,6 +55,120 @@ def test_create_tag_event
           type: 'CreateEvent',
           repo: { id: 42 },
           payload: { ref_type: 'tag', ref: 'foo' }
+        },
```

Comment:
@Suban05 what's the point of putting two test scenarios into one test method? It's definitely a bad practice, read this: https://www.yegor256.com/2018/12/11/unit-testing-anti-patterns.html Just create a new test methods, leaving the existing one untouched. Code duplication that may happen in this case is not a problem.

Reply:
@yegor256 I completely agree with you that this is a wrong approach. However, I couldn't make it work using a separate test. I tried using a different method, but every time, one of the tests failed, and each time by accident, I can't figure out why. I did it the same way as the first test, but each time in one of the tests, the `$fb` variable does not contain data, and the query returns an empty result.
Maybe I didn't understand how to use `Fastbase` correctly

Reply:
@Suban05 in such a situation you shouldn't blame yourself and waste time on experimenting. Just submit a new ticket, claiming that our testing system is not stable -- it doesn't allow you to add a new test method. I will fix this bug, you will earn points for reporting a ticket. Then, you continue with this PR.